engine/src/test/scala/cromwell/webservice/EngineStatsActorTest.scala

```diff
@@ -0,0 +1,60 @@
+package cromwell.webservice
+
+import akka.actor.{Actor, ActorRef, Props}
+import akka.testkit.{TestActorRef, TestProbe}
+import cromwell.core.TestKitSuite
+import cromwell.webservice.EngineStatsActor.{EngineStats, JobCount, JobCountQuery}
+import cromwell.webservice.EngineStatsActorTest.{BlockingFakeWorkflowActor, FakeWorkflowActor}
+import org.scalatest.{FlatSpecLike, Matchers}
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+class EngineStatsActorTest extends TestKitSuite("EngineStatsActor") with FlatSpecLike with Matchers {
+  behavior of "EngineStatsActor"
+
+  val replyTo = TestProbe()
+  val defaultTimeout = 100 millis
+
+  it should "return snakeyes with no WorkflowActors" in {
```

Comment:
You raise a good point but technically  we're talking about this: 
![snake_eyes_classic](https://cloud.githubusercontent.com/assets/961771/18621405/c5d6bdfe-7df0-11e6-9d9b-93838b84c1e0.png)